README.md

```diff
@@ -1,13 +1,3 @@
-# JHipster official "development box"
```

Comment:
Can you keep this line, which is the title plz ?

Reply:
Is it okay now?

Reply:
perfect, thanks @BhawaniSingh